Fix quantized ConvReLU3d test

xiaomengy opened the pull request:
Summary: Fix quantized ConvReLU3d test

Test Plan: buck test mode/dev-nosan //caffe2/test:quantized -- "conv"

Differential Revision: D18645717

jamesr66a reviewed (approved):
LGTM so long as tests pass

xiaomengy committed:
Fix quantized ConvReLU3d test (#30266)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/30266

Fix quantized ConvReLU3d test

Test Plan: buck test mode/dev-nosan //caffe2/test:quantized -- "conv"

Reviewed By: hl475

Differential Revision: D18645717

fbshipit-source-id: 1963a8415c3e9b18c88896b4c92733814f82e823
